### ISSUE TITLE
Constant "generating system asset: symbol_lists/<hash>" in build

### ISSUE THREAD

**Issue by juj** [emscripten-core/emscripten#18622]
After updating recently, I keep getting these prints
```
cache:INFO: generating system asset: symbol_lists/4b1c671fb19fde79292783da0799997fdd1b96c4.txt... (this will be cached in "C:\emsdk\emscripten\main\cache\symbol_lists\4b1c671fb19fde79292783da0799997fdd1b96c4.txt" for subsequent builds)
cache:INFO:  - ok
```
in my builds, which looks like unnecessary noise. Can those be avoided?

**Comment by sbc100** [comment]
Yes, we should try to do that.   We have another user report issues with that too:  https://github.com/emscripten-core/emscripten/issues/18607

I wonder if we should just turn of all "INFO:" messages by default.

**Comment by sbc100** [comment]
Specifically regarding those symbol_lists, they do get cached so should only happen the first time you build with a given set of flags.   We should still probably not print that info though.
